docs/spec/rans.decoding.md

```diff
@@ -26,7 +26,7 @@ void DecodeTaggedSymbols(num_values, num_components, out_values) {
   encoded_data                                                                        UI8[size]
   RansInitDecoder(ans_, &encoded_data[0], size, l_rans_base);
```

Comment:
Should `l_rans_base` here also be replaced with constant value?

Reply:
yes, will update now.